### DISCUSSION TITLE
Google Poly Setup

### DISCUSSION THREAD

**Discussion by n8icus** [mozilla/hubs#3150] [Hubs Cloud Questions] [answered]
When setting up google poly, do I need to set up the Oauth credentials as well or just use the api key? I'm able to explore the poly library in Spoke, but when I try to bring anything into the scene I get a 500 error from poly.

**Comment by robinkwilson** [comment, marked as answer]
Only the API key. Make sure it's saved in the admin panel.

If that isn't working, it's probably a bug. Do you have any console logs or info other than the 500 error?

Making a bug on [hubs-cloud repo issues](https://github.com/mozilla/hubs-cloud/issues). Make sure to specifically mention the Google Poly API key isn't working.

**Reply by n8icus** [reply]
Alright. I'll try to pull some console logs for you, and post a bug report. Could this have anything to do with CORS settings or any other settings on the server?

**Reply by n8icus** [reply]
I went through the Google Poly setup process again and I think I must have done it wrong before. Everything works now with the new credentials. Thank you for the help.
